src/CheckBox/CheckBox.test.jsx

```diff
@@ -5,12 +5,7 @@ import CheckBox from './index';
 
 describe('<CheckBox />', () => {
   it('attributes are set correctly', () => {
-    const wrapper = mount(
-      <CheckBox
-        name="checkbox"
-        label="check me out!"
-      />,
-    );
+    const wrapper = mount(<CheckBox name="checkbox" label="check me out!" />);
```

Comment:
I can definitely see how all-on-one-line is better per airbnb but *sigh* I really liked the old syntax...